Update guidance (#29)
worldbank/wb-reproducible-research-repository: guidance_note_wb.md

# Guidance Note for World Bank Staff and Consultants on Reproducible Publications

To enhance the credibility, transparency, and impact of its analytical products, the World Bank’s Development Economics Vice Presidency (DEC) launched the [Reproducible Research Repository](https://reproducibility.worldbank.org/). 

> The repository documents the data and code underlying Bank analytics and provides reproducibility packages, complete sets of instructions and materials required to regenerate the results of specific publications.  

Reviewer: ```suggestion
The repository documents the data and code underlying Bank analytics and provides reproducibility packages, complete sets of instructions and materials required to regenerate the results of specific publications.  As of September 2025, Policy Research Working Papers are required to produce a reproducibility package (if empirical) and all other publications are highly encouraged to do so. Publications with a reproducibility package are distinguished by a Reproducible Research badge on the front cover.
```

By making these packages available, the repository allows users to scrutinize the underlying data and methodological choices, reproduce results, and build on existing research. 
To ensure reliability, all reproducibility packages are verified for completeness and functionality before they are posted.  

Reproducibility verification ensures that a reviewer can regenerate exactly the outputs (statistical tables and data visualizations) presented in a publication by running the reproducibility package provided by the authors. 
In this process, the reviewer checks that the package is:  

- **Complete**: able to generate every output in the manuscript from the original data  
- **Stable**: producing identical results each time it is executed  
- **Consistent**: reproducing tables and figures that match those in the publication  

> This is not a review of the accuracy or quality of the code, the data, or the methods applied, or the validity of the research itself.  

Reviewer: ```suggestion
This is **not** a review of the accuracy or quality of the code, the data, or the methods applied, or the validity of the research itself.  
```

Detailed guidelines on the process for submission, verification, and publication of reproducibility packages are included below. World Bank staff and consultants can address any questions to the WB Reproducibility team at [reproducibility@worldbank.org](mailto:reproducibility@worldbank.org). 
Please review our FAQs [here](https://worldbank.github.io/wb-reproducible-research-repository/reproducibility_FAQs.html) as well.  

## Reproducibility package submission

> To begin, authors must complete the [Reproducibility Verification Request Form](https://survey.wb.surveycto.com/collect/prwp_reproducibility?caseid=) and submit their package through a GitHub repository, a OneDrive folder, another cloud-based sharing service, or, for small datasets, a compressed folder sent directly by email to reproducibility@worldbank.org. 

Author: The instructions are included in the form. The forma asks for the link to the reproducibility package or to share it with Mer/Luise. So their emails are already included in the form

The package should follow the [Reproducibility Package Checklist](https://github.com/worldbank/wb-reproducible-research-repository/blob/main/reproducibility_package_checklist.md). 
It needs to clearly document the data, code, and steps used to generate the paper’s results. 
A well-structured package typically organizes data, code, and outputs in separate folders, includes a README file, and provides a main script that specifies the order in which the code should be run. 
Supporting resources can be found [here](https://worldbank.github.io/wb-reproducible-research-repository/resources.html).

### Documenting Data  

A reproducibility package should always begin from documented data sources, often referred to as *source data*. Source data is a citable dataset that is properly documented, either publicly accessible or archived in a World Bank internal catalog, and serves as the starting point for reproducibility.

> For each external dataset used, authors are expected to provide a full data citation. This should include a URL for publicly available datasets, or clear instructions on how to compile or request access for datasets that cannot be redistributed, or require purchase or human approval.

Reviewer: ```suggestion
For each dataset used, authors are expected to provide a full data citation. This should include a URL for publicly available datasets, or clear instructions on how to compile or request access for datasets that cannot be redistributed, or require purchase or human approval.
```

Reviewer: should we link to AEA guidance on citation? could say something like "See guidance from a consortium of data editors at top economics journals here: https://social-science-data-editors.github.io/guidance/Guidance/addtl-data-citation-guidance.html"

> Any original data generated for the publication that is owned by the World Bank **must** also be deposited in one of the Bank’s official repositories: 

Reviewer: ```suggestion
Any original data generated for the publication that is owned by the World Bank **must** be deposited in one of the Bank’s official repositories (a copy of the data may be shared with the Reproducibility Team directly, to facilitate the verification process): 
```

In the case of confidential and/or restricted-access data, the authors have the following options which may be used as appropriate, these are listed in order of preference:

1. **Non-Disclosure Agreement (NDA)**  
   Reviewers sign an agreement stating that they will access the data only for reproducibility checks and will not re-use it. 
   Where needed, reviewers may request data directly from the provider under agreed terms.  

2. **Virtual Reproducibility Verification**  
   The reviewer observes the author running the package in a clean environment during a virtual meeting. 
   Once confirmed, the author shares the log and output files for detailed comparison with the manuscript. 
   See detailed protocol [here](https://worldbank.github.io/wb-reproducible-research-repository/guidance/virtual_reproducibility_verification_protocol.html).  

3. **Synthetic Data**  
   Authors provide fake but structurally similar data in the public package. 

>    The reviewer verifies the package with the actual data through one of the secure methods above.  

Reviewer: ```suggestion
   The reviewer verifies the package with the actual data through one of the secure methods above, and additionally verifies that the package runs with the synthetic data. The published version of the package includes the synthetic data, so users can interact directly with the code.
```

- A comparison report documenting any differences between the datasets used by the authors and the datasets currently available (applicable for Limited Access or Accessible data types)  

Every reproducibility package will be assigned a DOI to support discovery, cross-linking, and citation.  

### Treatment of Data in Published Packages  
Depending on access restrictions, data in the reproducibility package will take one of the following forms:  

1. **Data already published (internally or externally):**  
   The Data Availability Statement (DAS) links to the data with access instructions and version details. 
   If permitted, the data may also be included directly in the package, which is recommended when no DOI or stable link exists. ch is recommended when no DOI or stable link exists.  

2. **Proprietary or confidential data:**  
   The DAS explains how third parties may access the data. 
   Input data will not be included, though derivative datasets may be shared if allowed. 
   Data available to Bank staff should also be deposited in the internal catalogue and linked in the DAS.  

> 3. **World Bank–owned unpublished data:**  

Reviewer: This isn't very clear. I'm not sure how it's different from case 1. maybe this is the case of forthcoming or embargoed data, where we still publish the package?
